core/src/alpenglow_consensus/voting_loop.rs

```diff
@@ -504,6 +504,12 @@ impl VotingLoop {
                 new_root, &e
             );
         }
+        if let Some(config) = bank_notification_sender {
```

Comment:
might be nice to add a comment here similar to what you have in the PR description